src/frontend/js/widgets/SyllabusCourseRunsList/components/CourseProductItem/index.tsx

```diff
@@ -50,7 +50,13 @@ const messages = defineMessages({
     id: 'components.CourseProductItem.noSeatsAvailable',
   },
   nbSeatsAvailable: {
-    defaultMessage: '{nb} seat(s) available',
+    defaultMessage: `{
+nb,
+plural,
+=0 {No seat}
+one {1 seat}
+other {# seats}
+} remaining`,
```

Comment:
Sorry I realized there is a typo.

```suggestion
=0 {No remaining seat}
one {Last remaining seat!}
other {# remaining seats}
}`,
```